Source/WebCore/rendering/RenderFlexibleBox.cpp

```diff
@@ -1731,10 +1731,48 @@ LayoutUnit RenderFlexibleBox::staticMainAxisPositionForPositionedChild(const Ren
     return offset;
 }
 
-LayoutUnit RenderFlexibleBox::staticCrossAxisPositionForPositionedChild(const RenderBox& child)
+LayoutUnit RenderFlexibleBox::staticCrossAxisPositionForPositionedChild(const RenderBox& child) const
 {
     auto availableSpace = availableAlignmentSpaceForChild(crossAxisContentExtent(), child);
-    return alignmentOffset(availableSpace, alignmentForChild(child), 0_lu, 0_lu, style().flexWrap() == FlexWrap::Reverse);
+    auto alignContentDistribution = style().resolvedAlignContentDistribution(contentAlignmentNormalBehavior());
+    bool isReversed = style().flexWrap() == FlexWrap::Reverse;
+
+    // This lambda computes static positioning given a flex container's align-content property.
+    // Since static position of absolutely-positioned children is given by positioning the
+    // child on a hypothetical flex line and then that flex line shrinks-to-fit the child,
+    // we can simply compute where the flex line should be and return that as the child's static
+    // position.
+    // We specifically do not handle align-content: normal or align-content: stretch (same thing) here
+    // since that behavior is the same as in single-line flex containers. Note, this is not what the spec
+    // calls for when availableSpace is negative but it's what browsers have been shipping.
+    // Until the spec wording changes to match behavior then we are ignoring the spec for
+    // that one case in the name of web compatibility.
```

Comment:
Do we have a bug for this? If not, it'd be nice to have one in WebKit, so that future spec/interop discussions could refer to the different bug trackers.

You could even add there a complete description of the issue there and just include the link to the issue here mentioning that we don't follow the spec in this particular case.

Reply:
That's a great point. I wrote up a CSSWG issue but didn't make a corresponding bug in our bug tracker. I'll do that and then add the links here.